### DISCUSSION TITLE
Need help with HammerDB against sql server 2016

### DISCUSSION THREAD

**Discussion by amg1881** [TPC-Council/HammerDB#728] [General]
I have installed the latest hammerdb version 4.11 x64 on a windows server 2019. I’m able to build the schema and database tpcc with data loaded completes fine. The thing is that after that all icons are greyed out and I can’t open/choose the options of the Virtual users. I’m not able to start a test.  Can you help?
![image](https://github.com/user-attachments/assets/ec70690b-0fd2-4a8a-a601-c634b269a538)
 For your reference I have installed odbc  driver 17&18 for SQL server. I also tried running an older version of Hammerdb like 11.4 but no luck. I ran out of ideas. I also teied running hammerdb on Windows Server 2016 but same. Does anyone know what should I check and try?

**Comment by amg1881** [comment]
When clicking on the green “run virtual users” it says
![image](https://github.com/user-attachments/assets/5d6303b0-a5ae-4f67-8293-2cef99b10577)

**Reply by sm-shaw** [reply]
The quick start in the documentation is here https://www.hammerdb.com/docs/ch02.html. You are trying to re-run the schema build. 
Press the red stop button to close the virtual users down after the schema build. Then go to the driver script options once you press OK this will load the driver script. (or choose load to load the driver script without changing options) Then create the virtual users and run.

**Comment by amg1881** [comment]
Thank you very much. This resolved the issue. 

    On Thursday, July 25, 2024 at 11:47:58 AM GMT+1, Steve Shaw ***@***.***> wrote:  
 
 


The quick start in the documentation is here https://www.hammerdb.com/docs/ch02.html. You are trying to re-run the schema build.
Press the red stop button to close the virtual users down after the schema build. Then go to the driver script options once you press OK this will load the driver script. (or choose load to load the driver script without changing options) Then create the virtual users and run.
